tests/utils.ts

```diff
@@ -18,7 +23,9 @@ export const test = baseTest.extend<{ context: BrowserContext }>({
   context: async ({ context }, use) => {
     await context.addInitScript(() =>
       window.addEventListener('beforeunload', () =>
```

Comment:
suggestion (nice to have) - we can create this variable once instead of writing `as CollectIstanbulCoverageWindow` multiple times
```suggestion
      const w: CollectIstanbulCoverageWindow = window
      w.addEventListener('beforeunload', () =>
```